enclave/core/CMakeLists.txt

```diff
@@ -321,7 +321,7 @@ enclave_compile_definitions(
   $<BUILD_INTERFACE:OE_API_VERSION=2>)
 
 if (HAS_QUOTE_PROVIDER)
-  enclave_compile_definitions(oecore PUBLIC OE_LINK_SGX_DCAP_QL)
+  enclave_compile_definitions(oecore PUBLIC OE_HAS_SGX_DCAP_QL)
```

Comment:
Just curious, is there a way to determine whether the dcap library is loaded or not during the runtime? I'm asking because `OE_HAS_SGX_DCAP_QL` is still used at compile-time.

Reply:
We treat inability to load sgx_dcap_ql as a catastrophic error and terminate the application:
https://github.com/openenclave/openenclave/blob/ea85ec5403042e3937d2f531957e4e04a25bdb8a/host/sgx/sgxquote.c#L117

The expectation is that quoting is not requested in platforms that don't have sgx_dcap_ql.

Reply:
Thanks for the clarification!